### DISCUSSION TITLE
Github ssl connect error which gets resolved by taking session on the Linux agent box via Jenkins integration

### DISCUSSION THREAD

**Discussion by ishnasaran** [community/community#114174] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

 SSL connect issue for git requires physical session on the box.
This issue happens once or twice a week from Jenkins the ssl connect issue t github repo comes and the resolution to this issue is taking a physical connection to this box and nothing else. Can someone please help what is required to do. We are getting this issue from more than a month now.

Attaching the error screenshot.
<img width="772" alt="error" src="https://github.com/community/community/assets/131233778/2f1ca97c-b7a0-464d-bdf2-923367b2f642">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ishnasaran ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
